Classes/Outlet/Pipe/EmailPipe.php

```diff
@@ -155,7 +184,11 @@ public function conduct($data)
      */
     protected function prepareEmail($data)
     {
-        $body = $this->getBody();
+        if ($this->getBodySection() !== null) {
+            $body = $this->view->renderStandaloneSection($this->getBodySection(), $data);
```

Comment:
You could pass the third "$optional" argument `true` here to avoid errors with missing sections. I'm not sure if that makes sense in context of the feature.